### DISCUSSION TITLE
「开源之史」系列之十四/四：难辨真伪 —— 人人都可以开源许可下的困局 - 「开源之道」

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#70] [Announcements]
# 「开源之史」系列之十四/四：难辨真伪 —— 人人都可以开源许可下的困局 - 「开源之道」

https://opensourceway.community/posts/history-of-open-source/02-10-future-and-more-challenge/

**Comment by ZoomQuiet** [comment]
#是也乎,(￣▽￣)

也不能说摆烂, 只是无奈,

木兰协议, 俺理解其最大意义就在是中文的,
我们毕竟是五常之一, FLOSS/自由|开源软件 许可证以往都是英文版本为正式版本,
值得出个中文版本的, 至少对于普通人阅读, 减少了阻碍;

从 GPL -> BSD 这个光谱

当年的态度是这样的:
![zeuux](http://s5.zoomquiet.top/080130-floss-license/img/080129-floss-taidu.png)

现在其实态度没变,
行为上佛系很多,
毕竟, FLOSS/自由|开源软件 项目托管的各种平台可都是商业公司在支撑着;

嘦没作出明显的恶意侵害, 都可以, 没关系,
毕竟我们自己运转的很好.

但是, 这不是一个 Evangelist 应该的态度,
得在事情没有腐败前, 想明白应该如何来....

**Reply by lijiangsheng1** [reply]
我的立场是自由软件立场，最左侧的～
